kurl_util/cmd/pvmigrate/main.go

```diff
@@ -37,8 +38,19 @@ func main() {
 	flag.BoolVar(&opts.VerboseCopy, "verbose-copy", false, "show output from the rsync command used to copy data between PVCs")
 	flag.BoolVar(&opts.SkipSourceValidation, "skip-source-validation", false, "migrate from PVCs using a particular StorageClass name, even if that StorageClass does not exist")
 	flag.BoolVar(&skipFreeSpaceCheck, "skip-free-space-check", false, "skips the check for storage free space prior to running the migrations")
+	flag.BoolVar(&skipPvAccessModeCheck, "skip-pv-access-mode-check", false, "skips the volume access modes checks prior to running the migrations")
+	flag.BoolVar(&dryRun, "dry-run", false, "run validation checks without running the migrations")
+	flag.BoolVar(&printVersion, "version", false, "Print the version of the client")
+	flag.IntVar(&opts.PodReadyTimeout, "pod-ready-timeout", 90, "length of time to wait (in seconds) for volume validation pod(s) to go into Ready phase")
+
 	flag.Parse()
 
+	if printVersion {
+		fmt.Printf("Running pvmigrate build:\n")
+		version.Print()
+		os.Exit(0)
+	}
```

Comment:
It seems that we need another subcommand instead of it.

Reply:
is there a reason NOT to print the version?

We want to be able to figure out the version from the logs a customer sends us showing an error

Reply:
@laverya I added this to make the console messages more readable when the installer is doing an upgrade from rook to openebs. Basically, I wanted to seamlessly embed `pvmigrate` in the installer and abstract its usage.

Unfortunately, kURL console logs are rarely provided by the customer.

Reply:
@laverya I will push a change to print the `pvmigrate` change.